packages/lib/sdk/src/configuration/migratePluginsToConfig.js

```diff
@@ -0,0 +1,42 @@
+import fs from 'fs/promises';
+import path from 'path';
+import { getEvidenceConfig } from './getEvidenceConfig.js';
+import { updateEvidenceConfig } from './updateEvidenceConfig.js';
+import { log } from '../logger/index.js';
+import { projectRoot } from '../lib/projectPaths.js';
+import { getEvidenceConfigLegacy } from './getEvidenceConfig.legacy.js';
+import merge from 'lodash.merge';
+import { EvidenceConfigSchema } from './schemas/config.schema.js';
+
+/** @returns {Promise<boolean>} */
+export const migratePluginsToConfig = async () => {
+	const pluginStat = await fs
+		.stat(path.join(projectRoot, 'evidence.plugins.yaml'))
+		.then(() => true)
+		.catch(() => false);
+	if (!pluginStat) {
+		log.warn('No evidence.plugins.yaml file found in project directory, no action was taken.');
+		return false;
+	}
+
+	/** @type {import("./schemas/config.schema.js").EvidenceConfig} */
+	let config = EvidenceConfigSchema.parse({ plugins: {} });
+	const legacyConfig = getEvidenceConfigLegacy(true);
+	try {
+		config = getEvidenceConfig(EvidenceConfigSchema, false);
+	} catch {
+		/* ignore errors */
+	}
+
+	config.plugins = merge(legacyConfig?.plugins ?? {}, config.plugins);
+	await updateEvidenceConfig(config);
+
+	// if evidence.plugins.yaml exists in the project directory, delete it
+	try {
+		await fs.unlink(path.join(process.cwd(), 'evidence.plugins.yaml'));
+	} catch (e) {
+		// do nothing
+		log.debug('Error Encountered Deleteing evidence.plugins.yaml', [e]);
```

Comment:
Why `[e]` and not just `e`?
```suggestion
		log.debug('Error Encountered Deleteing evidence.plugins.yaml', e);
```